ci/plugins/mzcompose/hooks/command

```diff
@@ -80,4 +80,7 @@ fi
 
 ci_uncollapsed_heading ":docker: Running \`bin/mzcompose --find \"$BUILDKITE_PLUGIN_MZCOMPOSE_COMPOSITION\" run ${run_args[*]}\`"
 
+docker run --rm -t amd64/ubuntu uname -m
```

Comment:
A comment here would be appreciated . those command lines have one too many unusual arguments

Reply:
Oops, that was just me experimenting to see if qemu-static was working with docker, which was a requirement for this whole test.